### ISSUE TITLE
Moot normalization

### ISSUE THREAD

**Issue by sindresorhus** [zeke/electron-shortcut-normalizer#1]
>  converts CmdOrCtrl modifier to CommandOrControl

Electron already supports `CmdOrCtrl` as an alias for `CommandOrControl`, so there's really no point in doing this normalization.

https://github.com/electron/electron/blob/master/docs/api/accelerator.md#available-modifiers:

> CommandOrControl (or CmdOrCtrl for short)

**Comment by zeke** [comment]
Good point. That normalization is really just there so we have a common string format to normalize to a specific platform:

https://github.com/zeke/electron-shortcut-normalizer/blob/4c1240c1d33211eaaf14e41f68f3da591ed3f668/index.js#L17

Feel free to submit a PR, otherwise I will get to it when I can.
